apps/tester-federation/package.json

```diff
@@ -9,13 +9,15 @@
     "start:miniapp": "react-native webpack-start --webpackConfig webpack.config.mini-app.mjs --port 8082"
   },
   "dependencies": {
-    "react": "18.2.0",
-    "react-native": "0.74.3",
+    "@callstack/repack": "workspace:*",
+    "@module-federation/enhanced": "^0.3.2",
+    "@module-federation/runtime": "^0.3.2",
```

Comment:
You don’t need runtime installed unless. Enhanced has it and injects. Unless you’re using runtime outside of the build scope.

Reply:
Oh yeah, probably just a leftover before I realised I don't need to do runtime shared deps since in webpack MF entry was first 👌